include/dwg.h

```diff
@@ -9037,10 +9037,11 @@ typedef enum DWG_SECTION_TYPE_R13
   SECTION_HEADER_R13 = 0,
   SECTION_CLASSES_R13 = 1,
   SECTION_HANDLES_R13 = 2,
-  SECTION_OBJFREESPACE_R13 = 3, /* including the 2ndheader */
-  SECTION_TEMPLATE_R13 = 4,
-  SECTION_AUXHEADER_R2000 = 5,
-  SECTION_THUMBNAIL_R13 = 6,
+  SECTION_OBJFREESPACE_R13 = 3,
+  SECTION_SECOND_HEADER_R13 = 4,
```

Comment:
Nope this confuses our section numbering even more. I'd like it keeping attached to the ObjFreeSpace

Reply:
I need to look again.